javascript/javascript2/week1/shortest-word-in-an-array.js

```diff
@@ -1,14 +1,14 @@
 const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
 
-function shortestWord(array) {
+function shortestWord(words) {
   let shortest = '';
 
-  for (let element of array) {
+  for (let word of words) {
     if (
-      typeof element === 'string' &&
-      (shortest === '' || element.length < shortest.length)
+      typeof word === 'string' &&
+      (shortest === '' || word.length < shortest.length)
```

Comment:
No need to check for empty string
```suggestion
      ( word.length < shortest.length)
```